requirements_dev.txt

```diff
@@ -20,5 +20,5 @@
 
 # used for development (typing)
     mypy >= 1.15.0
-    types-paho-mqtt >= 1.6.0
+    types-paho-mqtt >= 1.6.0                     # no longer required when we use paho-mqtt 2.x
```

Comment:
why did you remove this comment? (a 3rd PR, unrelated to the other 2?)

Reply:
I added the comment, in fact. There is a PR in ramses_rf to bump paho_mqtt to >= 2.1.0 and that should allow type hints without this stub. For now, just a marker here.